### ISSUE TITLE
django 1.3 STATIC_URL

### ISSUE THREAD

**Issue by munhitsu** [dziegler/django-css#28]
Django 1.3 introduces STATIC_URL as a standard approach for app static files.
So from now on there are two paths for not dynamic files:
MEDIA_URL - more like uploaded files
STATIC_URL - statics delivered with app

At this point there are two issues with django-css.
1. defeulting to context MEDIA_URL
I guess it should be enough jus to remove templatetags/compress.py lines:
        if 'MEDIA_URL' in context:
            media_url = context['MEDIA_URL']
        else:
1. During development STATIC_ROOT is often empty as it is being dynamically filled by static app. A standard practice is to have all files collected to STATIC_ROOT only on development

It might be useful to either document this limitation or get real static file path while in development mode (runserver)

**Comment by hedberg** [comment]
I made a pull request regarding the media url issue earlier. You can have a look here https://github.com/dziegler/django-css/pull/27

**Comment by munhitsu** [comment]
Works perfect. Thanks!

**Comment by twoolie** [comment]
i dont know about you guys, but i store site-wide sass files in STATIC_ROOT and now these files are not found by django-css. They are however served perfectly by django static files.

django-css needs to considder files in STATIC_ROOT before any files in STATICFILES_DIRS

**Comment by xster** [comment]
Django-css's copy of django-compressor is old in general. Django-compressor took care of this in recent versions

**Comment by twoolie** [comment]
is anyone merging django-compressor into django-css regularly/semi-regularly?

**Comment by hedberg** [comment]
You might wanna take a look at https://github.com/jezdez/django_compressor/. It now has a precompiler filter with which you can add support for external commands (sass, coffescript etc). Works well in my first test and have support for Django 1.3 / staticfiles.

**Comment by ghost** [comment]
Yes, it works really well in my tests. Except one small problem where I can't get it to find my static files in my app. Other than that there's 0 problems. :)

**Comment by ghost** [comment]
I tracked down the bug I spoke of above [here](https://github.com/jezdez/django_compressor/issues/51), and have the fix.
